cmd/lakectl/cmd/log.go

```diff
@@ -80,12 +99,33 @@ var logCmd = &cobra.Command{
 					After:   pagination.NextOffset,
 				},
 			}
-			Write(commitsTemplate, data)
+			if dot {
```

Comment:
Can we extract the code to a function as use `data` as it captures the current result information.
I expected to see here something like:

```go
tmpl := commitsTemplate
if dot {
    tmpl = commitsDotTemplate
} 
Write(tmpl, data)
```

Or

```go
if dot {
    writeCommitsDot(branchURI.Repository, data)
} else {
    Write(commitsTemplate, data)
}

```

Reply:
I prefer to avoid holding the entire commit graph in memory when possible

Reply:
Not suggesting the load the complete log - we process each call results - same as Write(commitsTemplate, data). we can extract method, have the code outside the loop (the using of the template engine is optional)